README.md

```diff
@@ -1,363 +1,209 @@
+Periodically, the code and the content changes. Make sure you're staying
+up-to-date with these commands:
 
     git pull origin master
-    git submodule update
-
-## Development
-
-To do local development, there are many services to start. The simplest
-is to use `nf` which is a dev dependency that gets installed by the
-root `package.json` and executed like this:
-
     yarn
+    yarn clean
     yarn start
 
-That will start a React dev-server at `http://localhost:3000`. If you
-change any of them files in `client/src` it will reload and refresh your
-browser. If you edit any of the Markdown files in `stumptown` it will
-repackage that file and refresh your browser too.
-
-Note! We hope to use, as an alternative to `nf`, `docker-compose`.
-See [https://github.com/mdn/yari/issues/23](https://github.com/mdn/yari/issues/23)
-But even then, using `docker-compose` should and will be optional.
-
-## Contributing
-
-Open two terminals. In one, run (this will take a little time the first time):
-
-    yarn workspace server start
-
-In another terminal:
+These are also good steps to always take when you embark to make a change.
+Then, the only extra command needed is `git checkout -b my-new-branch`
+(or however you prefer to create new `git` branches)
 
-    yarn workspace client start
+## License
 
-Now you should have two servers:
+All source code is MPL-2.0.
 
-1. [http://localhost:3000](http://localhost:3000) (open this in your browser)
+For content, see [issue #408](https://github.com/mdn/yari/issues/408).
 
-2. [http://localhost:5000](http://localhost:5000)
+## How it works
 
-Note that when you run the `React` dev server (on `localhost:3000`) it
-depends on the files built by `stumptown` and consequently built by
-the `cli`. You can now hack on the key `React` components and just refresh
-the browser to see the effect immediately. If you want re-build the
-content made available to the `React` components, open another terminal
-and run:
+Yari is multiple things but at its core is the MDN content as `index.html`
+files, in `git`, that contain the metadata (as front-matter) and
+the bulk of the document.
 
-    yarn build
+The builder converts these "source files" into "build files" using a CLI tool
+that iterates over the files, builds the HTML, and lastly packages it up
+with the front-end code. Ready to be served as static file.
 
-To re-run any of the installation and build steps you can, at any time,
-run:
-
-    yarn
-
-### Testing production builds
+## Development
 
-Suppose that you're working on a feature or bug that only relates to the
-files being compiled for production use, here's how you'd do that:
+First of all, development on `yari` can mean the source code (e.g. the
+styling of the header) or it can mean the content, since it's all one
+repo. This document doesn't distinguish between the two. In the future we
+might expand with more documentation specifically for contributing on the
```

Comment:
'for contributing content exclusively.'

Reply:
That one I don't get. I like the "contributing ON THE content" better because it feels like that includes editing better whereas "contributing content" feels like *adding* content.

Reply:
ahh, ok, in that case, it probably should be 'contributing to the content'

Reply:
Got it!